AzureCalling/Controllers/IntroViewController.swift

```diff
@@ -6,7 +6,7 @@
 import FluentUI
 import UIKit
 
-let learnMoreURL = "https://aka.ms/ACS-CallingSample-iOS"
+let learnMoreURL = "https://azure.microsoft.com/services/communication-services/"
```

Comment:
Are we switching the from the aka.ms link for a specific reason?

Reply:
I've actually never made one before, but if it's possible to update where it points to, it might be better to keep it an aka.ms link. That way we don't need to do a code change to update the link in the future.

Reply:
I like your thought, but for me the aka.ms tool is in an infinite-redirect-login loop.
We're switching the link away from referencing the code sample (as you've already got that running to see this page in the app).

Reply:
Alright, got in with edge rather than Safari. Making an aka.ms link